src/Routing/DynamicRouter.php

```diff
@@ -41,7 +41,15 @@ class DynamicRouter extends BaseDynamicRouter
      * key for the request attribute that contains the template this document
      * wants to use.
      */
-    const CONTENT_TEMPLATE = 'contentTemplate';
+    const CONTENT_TEMPLATE = 'template';
+
+    /**
+     * RoutingBundle 1.* put the template into the request attributes as "contentTemplate".
+     *
+     * This name is deprecated in favor of "template" for consistency with
+     * FrameworkBundle and will be removed in RoutingBundle 3.
+     */
+    const BC_CONTENT_TEMPLATE = 'contentTemplate';
```

Comment:
I don't think we have to use a constant for the BC version, it'll only cause more conflicts (people shouldn't start using this constant)